tests/test_traverse_project.py

```diff
@@ -510,6 +512,66 @@ class TraverseProjectVector:
             ".venvs/another-venv/lib/python3.11/site-packages",
         },
     ),
+    #
+    # Test invalid 'ignores':
+    #
+    TraverseProjectVector(
+        "empty_ignore_pattern__raises_IgnoreRuleMissing",
+        "empty",
+        code=set(),
+        deps=set(),
+        pyenvs=set(),
+        ignores={"", "\t", "    "},
+        expect_raised=IgnoreRuleMissing,
+    ),
```

Comment:
The error is caused by the tab on the second line? or the general lack of ignore patterns?

Reply:
The general lack of ignore patterns. The intention here was to test that a blank line is considered empty whether or not it contains extra whitespace. These should be three different tests, I think.